sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/components/CreateNotificationPresenter.groovy

```diff
@@ -43,4 +43,9 @@ class CreateNotificationPresenter implements CreateNotificationOutput {
         //ToDo failNotification should be passed into dedicated logging file on executing server
         println(notification)
     }
+
+    void sendEmailNotifications(){
+        EmailGenerator emailGenerator = new EmailGenerator()
+        emailGenerator.sendEmails(notifications)
+    }
```

Comment:
For readingflow and understanding this would be what I expect. It feels artificial to move the notifications through the dependency manager. I would expect them to be provided by the use case :)

```suggestion
    void sendEmailNotifications(List notifications){
        emailGenerator.sendEmails(notifications)
    }
```

Reply:
However this means that the retrieved notifications will have be exposed to the outermost layer of the application

Reply:
Nevermind this is clarified